eZ/Publish/Core/Helper/FieldsGroups/ArrayTranslatorFieldsGroupsList.php

```diff
@@ -45,4 +46,13 @@ public function getDefaultGroup()
     {
         return $this->defaultGroup;
     }
+
+    public function getFieldGroup(FieldDefinition $fieldDefinition): string
+    {
+        if (empty($fieldDefinition->fieldGroup)) {
+            return $this->getDefaultGroup();
+        }
+
+        return $fieldDefinition->fieldGroup;
+    }
```

Comment:
We should to add test case here \\eZ\\Publish\\Core\\Helper\\Tests\\FieldsGroups\\ArrayTranslatorFieldsGroupsListTest

Reply:
Added tests coverage